alicloud/service_alicloud_cms.go

```diff
@@ -87,3 +87,16 @@ func (s *CmsService) ExtractWebhookFromJson(webhookJson string) (string, error)
 	}
 	return dat["url"].(string), nil
 }
+
+func (s *CmsService) DescribeSiteMonitor(keyword string) (*cms.DescribeSiteMonitorListResponse, error) {
+	listRequest := cms.CreateDescribeSiteMonitorListRequest()
+	listRequest.Keyword = keyword
+	raw, err := s.client.WithCmsClient(func(cmsClient *cms.Client) (interface{}, error) {
+		return cmsClient.DescribeSiteMonitorList(listRequest)
+	})
+	list := raw.(*cms.DescribeSiteMonitorListResponse)
+	if err != nil {
+		return nil, err
+	}
+	return list, nil
```

Comment:
This method should return one object. If there is no specified object, should return a not found error and you can refer to other service method.

Reply:
done